tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest>=3.8.2
+pexpect>=4.6.0
```

Comment:
Tests requirements are set as extra dependencies in the `setup.py` file:
https://github.com/deepmipt/DeepPavlov/blob/5b2132709ce51c370629a66b2296033dc5e73d99/setup.py#L57-L60